### PR TITLE
remove updateAptWithMicrosoftPkg call from cse main

### PR DESCRIPTION
**Pull request by ganeshkumar5699**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Egress tests in RP, restrict calls to packages.microsoft.com, during CSE execution. Removing `updateAptWithMicrosoftPkg` as that makes calls to packages.microsoft.com, in `cse_main.sh`.

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: e2e/e2e-scenario.sh [hunk 1]
```diff
@@ -75,7 +75,7 @@ az vmss create -n ${VMSS_NAME} \
     --vm-sku $VM_SKU \
     --instance-count 1 \
     --assign-identity $msiResourceID \
-    --image "microsoft-aks:aks:aks-ubuntu-1804-2022-q1:2022.02.01" \
```
Comment (ganeshkumar5699) [inline e2e/e2e-scenario.sh:78]: Updating this image, since it needs the newer VHD (with the updated apt packages).

Comment (ganeshkumar5699) [reply e2e/e2e-scenario.sh:78]: Otherwise Agentbaker e2e will fail, even though the behavior is correct

Comment (ganeshkumar5699) [reply e2e/e2e-scenario.sh:78]: Will likely change it to a different image before merging (GPU tests seem to have restrictions due to it too)

File: e2e/e2e-scenario.sh [hunk 2]
```diff
@@ -75,7 +75,7 @@ az vmss create -n ${VMSS_NAME} \
...
+    --image /subscriptions/8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8/resourceGroups/aksvhdtestbuildrg/providers/Microsoft.Compute/galleries/PackerSigGalleryEastUS/images/1804Gen2/versions/1.1658785995.106 \
```
Comment (alexeldeib) [inline e2e/e2e-scenario.sh:78]: please revert due to auth issues

Comment (alexeldeib) [reply e2e/e2e-scenario.sh:78]: err, gc issues? both? lol

**Review by alexeldeib — APPROVED** [review]

**Review by anujmaheshwari1 — APPROVED** [review]

**Pull request merged**